src/consumer.rs

```diff
@@ -37,6 +37,45 @@ impl<
     }
 }
 
+/// Continuously consumes message from a Queue
+///
+/// A consumer represents the [basic.consume](https://www.rabbitmq.com/amqp-0-9-1-quickref.html#basic.consume) AMQP command.
```

Comment:
It doesn't represent the command itself, but rather the stream of messages created by the command

Reply:
Thanks, updated the documentation.